src/ol/collection.js

```diff
@@ -82,7 +82,7 @@ ol.CollectionProperty = {
  */
 ol.Collection = function(opt_array) {
 
-  goog.base(this);
+  ol.Object.call(this);
```

Comment:
`goog.base` is used both in constructors and to call super class methods, which is quite confusing.

@ahocevar, are you refering to the `arguments` received by the `goog.base` function as used in https://github.com/google/closure-library/blob/master/closure/goog/base.js#L2460-L2465?

That one should be different from the `arguments` received by the `ol.Collection` constructor.